plugins/psv-extractor/src/psv.extractor.ts

```diff
@@ -8,45 +8,72 @@ import Papa, { ParseResult } from 'papaparse'
 
 export class PsvExtractor extends AbstractExtractor {
   private readonly _options: {
-    //add if needed
+    delimiter?: ',' | '\t' | '|' | ';'
+    dynamicTyping?: boolean
+    header?: boolean
+    skipEmptyLines?: boolean | 'greedy'
+    transform?: (value: any) => any | undefined
   }
 
   constructor(
     public event: Flatfile.UploadCompletedEvent,
     public options?: {
-      //add if needed
+      delimiter?: ',' | '\t' | '|' | ';'
+      dynamicTyping?: boolean
+      header?: boolean
+      skipEmptyLines?: boolean | 'greedy'
+      transform?: (value: any) => any | undefined
     }
   ) {
     super(event)
-    this._options = { ...options }
+    this._options = {
+      delimiter: '|',
+      dynamicTyping: false,
+      header: true,
+      skipEmptyLines: 'greedy',
+      transform: (value) => value || '',
+      ...options,
+    }
   }
 
-  public parseBuffer(fileContents: string): WorkbookCapture | undefined {
+  public parseBuffer(buffer: Buffer): WorkbookCapture {
     try {
+      const sheetName = 'Sheet1' // Set the sheet name
```

Comment:
⛳ This comment doesn't provide much value beyond what the code reads. Maybe we should remove it.